plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/packagematerial/JsonMessageHandler1_0.java

```diff
@@ -159,7 +159,10 @@ public String requestMessageForLatestRevision(PackageConfiguration packageConfig
 
     @Override
     public PackageRevision responseMessageForLatestRevision(String responseBody) {
-        return toPackageRevision(responseBody);
+        PackageRevision packageRevision = toPackageRevision(responseBody);
+        if( packageRevision == null) {
```

Comment:
I ran a [reformat and committed](https://github.com/gocd/gocd/commit/4c96d4dd8e3639ef7b50c6534c146ca91b51a711).